src/pages/experiments/[experimentId]/data-exploration/components/generic-gene-table/ListSelected.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Button, Select,
+} from 'antd';
+import { useSelector } from 'react-redux';
+
+const ListSelected = () => {
+  const selectedGenes = useSelector((state) => state.genes.selected);
+  const [listed, setListed] = useState(false);
+  console.log(selectedGenes, listed);
+  if (listed && !selectedGenes.length) setListed(false);
+  if (listed) {
+    return (
+      <>
+        <Button type='link' size='small'
+          onClick={() => { setListed(false) }}
+        >Hide selected</Button>
+        <Select
+          value={selectedGenes}
+          mode='multiple'
+          showArrow={false}
+          style={{ width: '100%' }}
+        />
+      </>
+    );
+  }
+  return (
+    <Button type='link' size='small'
+      onClick={() => { setListed(true) }}
+    >List selected</Button>
+  )
+}
```

Comment:
make it so that if the user clicks on `List Selected` and there are no selected genes, to show an empty space with no selected genes. At the moment, the button doesn't do anything if `selectedGenes` is empty, which is a usability issue.